lib/ClangImporter/ClangImporter.cpp

```diff
@@ -658,7 +658,7 @@ importer::getNormalInvocationArguments(
     }
   }
 
-  if (searchPathOpts.SDKPath.empty()) {
+  if (searchPathOpts.SDKPath.empty() && !triple.isOSFreeBSD()) {
```

Comment:
Why special case this for FreeBSD, not working?

Reply:
Yes, it excludes all system headers, causing e.g. the Glibc module to not work. I am unsure why this works on Linux, but not FreeBSD.

Reply:
You could try adding the ClangImporter flags ` -Xfrontend -dump-clang-diagnostics -Xcc -v` after removing this special case to dump what it's seeing and compare with linux, as I did with SR-14796.